manifests/dev/vsphere-7.0u2/supervisorcluster/k8s-1.17/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -178,6 +178,7 @@ apiVersion: v1
 data:
   "volume-extend": "true"
   "volume-health": "true"
+  "file-volume": "false"
```

Comment:
I see a change done for 1.17 and 1.19 version of k8s. Any reason why you are not doing a similar change for 1.18(https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/dev/vsphere-7.0u2/supervisorcluster/k8s-1.18/deploy/vsphere-csi-controller-deployment.yaml)?